modular_skyrat/modules/modular_reagents/code/modifiers/reagent.dm

```diff
@@ -0,0 +1,9 @@
+//*
+//REAGENT MODIFIERS - used for handling movespeed, actionspeed buffs, etc.
+//*
+//Ataraxydone - stasis chem, has a modifier which varies based on number of cycles in + a flat modifier if you're prone while it's in your system (atarxyprone)
```

Comment:
Documentation should always be using either `///` or `/* */` (for multiline ones), otherwise they won't be recognized by the autodoc function of the DM Lang server extension, or the AUTODOC workflow on the repo.